### DISCUSSION TITLE
x32-rack-mixer-psu-replacement/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#225] [Announcements]
# x32-rack-mixer-psu-replacement/

I bought a used unit off of Reverb and after about 75 days the unit began exhibiting these symptoms in the video. Neither Reverb nor the seller cared any so I began the hunt for a fix.

https://blog.ktz.me/x32-rack-mixer-psu-replacement/

**Comment by audiolabga** [comment]
I would like to Thank You for posting about Music Tribe Parts (Behringer, Tannoy, Midas, Turbosound and more).  We have been working hard over the last five years to stock items in the USA.  This is a great post for anyone needing to repair this model.  It is the number one failure of all models at this time.  
JR

**Reply by ironicbadger** [reply]
I'm honestly glad you had the part. I had to search high and low for it for several months to find stock.

**Comment by imholland** [comment]
Hi Alex,
I am having trouble seeing the "release" to depress ---
"The spade connectors have a little release on them if you slide back the plastic jacket and depress the release, they will pop right off.
Can you point me in the right direction?   The plastic jacket is back.

**Comment by audiolabga** [comment]
Try twisting them while pulling them with needle nose pliers
